test_requirements.txt

```diff
@@ -2,6 +2,7 @@ flake8==3.8.4
 mypy==0.790
 python-dateutil==2.7.5
 django-extensions
-freezegun==0.3.11
+freezegun==0.3.12
```

Comment:
0.3.11 is not compatible with python 3.8 - `time.clock()` function has been removed.

Reply:
(same version as our core, even though freezegun is at 1.1.0 right now)